core/templates/domain/collection/collection-node.model.ts

```diff
@@ -57,14 +57,9 @@ export class CollectionNode {
   }
 
   // Returns the title of the exploration represented by this collection node.
-  // This property is immutable. The value returned by this function is
-  // null if doesExplorationExist() returns false.
+  // This property is immutable.
   getExplorationTitle(): string {
-    if (this._explorationSummaryObject) {
-      return this._explorationSummaryObject.title;
```

Comment:
Can you please verify once that this model is never constructed with an empty collection node object, something like 
```typescript
{
  'exploration_id': null,
  'exploration_summary': null
}
```

Reply:
Exploration id is to be provided for creating this object.